tests/unit/docker/test_hub_build_level.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import pytest
+import mock
+from mock import Mock, patch
+
+from jina.docker.hubio import HubIO
+from jina.helper import yaml
+from jina.enums import BuildTestLevel
+from jina.peapods import Pod
+from jina.executors import BaseExecutor
+from jina.executors import AnyExecutor
+from jina.parser import set_hub_build_parser
+
+@pytest.fixture
+def mock_load_config():
+    return BaseExecutor
+
+def test_hub_build_level_pass(monkeypatch, mock_load_config):
+
+    monkeypatch.setattr(BaseExecutor, "load_config", mock_load_config)
+    args = set_hub_build_parser().parse_args(['path/hub-mwu', '--push', '--host-info', '--test-level', 'EXECUTOR'])
+
+    p_names, failed_levels = HubIO._test_build("jinahub/pod.dummy_mwu_encoder", BuildTestLevel.EXECUTOR, "sample/yaml.yaml", True)
```

Comment:
this is a completely misleading test, it tests nothing.
`_test_build` does not accept image as `str`, it accepts a `Docker Image Object`